docker-compose.yml

```diff
@@ -0,0 +1,4 @@
+pyvaultone:
```

Comment:
why do you need this docker compose? @vavieira10

Reply:
@LeoCBS Removed!